### DISCUSSION TITLE
Add more options for Auto Start time or allow custom input

### DISCUSSION THREAD

**Discussion by vadrile** [ppy/osu#31205] [Ideas] [answered]
I think 20 seconds is ideal for selecting a map after a game, though 10 seconds might suffice when there are already many maps in the queue.
Alternatively, it might be even better if users could set the time freely.
(I use machine translation.)

![スクリーンショット 2024-12-20 103837](https://github.com/user-attachments/assets/aa9b45b6-5eaa-4bc7-845b-058adac7120c)

**Comment by peppy** [comment]
I'd be fine with adding a 10 or 15 second option.

**Reply by vadrile** [reply]
Personally, I think 15 seconds might be just right if we add one more.
Thank you for considering it :D

**Reply by peppy** [reply, marked as answer]
Went with 10 instead. https://github.com/ppy/osu/pull/31381.
